data/tsv/egy_latn_phonemic.tsv

```diff
@@ -2051,12 +1995,6 @@ tr	t ɛ r
 ttj	t a t i j
 ttj	t ɛ t i
 tw	t uː
-tw.j	t uː ʔ i
```

Comment:
I suspect it's the symbols like "." that trigger the "Common" script - we'll need to modify the logic and extend the testcase to not filter those.

Reply:
I think that's correct. One thing to be aware of is we will get entries like the following in a scrape of English:
```
++	p l ʌ s p l ʌ s
@	i t ʃ æ t
```
Or in a scrape of Chinese:
```
🌶	l ä
🍋	s w ä n ⁵ ⁵
```
(All classed as Common)
I suppose we have to decide whether we want such entries in our scraped data, if we decide we don't then, at least for English, I think we'll need a more complex solution than folding Common into Latin.

Reply:
because `_detect_best_script_name()` returns a probability as well, I'd imagine that `++` would return 1.0 probability for 'Common' on this symbol and a value less than 1.0 for something like 'tw.j'. I didn't use that probability for anything in the script, but we might be able to use it to help inform classification? I'll play around with things and see if that's even true to begin with.